.gitignore

```diff
@@ -4,4 +4,7 @@ node_modules
 
 # Files created by build scripts
 test-es5
-lib
\ No newline at end of file
+lib
+
+# PhpStorm
```

Comment:
It's small thing, but I believe that folder is used by 'WebStorm' in general. Looks funny to talk about php in javascript library, you know :)